src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+const baseUrl = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones';
+
+const detailsUrl = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones/';
```

Comment:
```suggestion
const DETAILS_URL
  = 'https://mate-academy.github.io/phone-catalogue-static/api/phones/';
```
для назв таких констан зазвичай використовується верхній регістр